srcpkgs/python3-pytest-xdist/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-pytest-xdist'
+pkgname=python3-pytest-xdist
+version=2.5.0
+revision=1
+wrksrc="${pkgname#python3-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-setuptools_scm"
+depends="python3-pytest python3-pytest-forked python3-execnet"
+short_desc="Pytest plugin for distributed testing & loop-on-failures testing modes"
+maintainer="classabbyamp <dev@placeviolette.net>"
+license="MIT"
+homepage="https://github.com/pytest-dev/pytest-xdist"
+changelog="https://raw.githubusercontent.com/pytest-dev/pytest-xdist/master/CHANGELOG.rst"
+distfiles="https://github.com/pytest-dev/pytest-xdist/archive/refs/tags/v${version}.tar.gz"
+checksum=b0623f7b0e190175f46dbc939cc143ca49af2e5bdc70f2627c083b3ed1109f26
+# tests only work in tox
+make_check=no
```

Comment:
In #37172 checks just worked, despite a warning about tox being deprecated soon.

Reply:
it didn't run any tests, which is expected because the tests in xdist use pytest, which wasn't added to checkdepends: https://github.com/void-linux/void-packages/runs/6477482904?check_suite_focus=true#step:7:107